en/tools/config-catalog-generator-cc/data/enforcer/configs.json

```diff
@@ -447,6 +447,47 @@
       ],
       "exampleFile": "auth_header.toml"
     },
+    {
+      "title": "Mutual SSL",
+      "options": [
+        {
+          "name": "enforcer.security.mutualSSL",
+          "required": false,
+          "description": "Configurations related to Mutual SSL",
+          "params": [
+            {
+              "name": "certificateHeader",
+              "type": "string",
+              "required": true,
+              "default": "X-WSO2-CLIENT-CERTIFICATE",
+              "description": "Header name which client certificate coming from the downstream client"
```

Comment:
The header name from which the client certificate is coming is from the downstream client.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7065/commits/4ac3b72b3278ab8265e5f4635499d5ea37db0c04